src/Components/Molecules/cart-drop-down/cart-drop-down.tsx

```diff
@@ -0,0 +1,18 @@
+import "../cart-drop-down/cart-drop-down.scss";
+import { Cart } from "../../../data";
+
+const CartDropDown = (props: Cart) => {
+  return (
+    <div className="cart">
+      {props.cart.map((post, index) => {
```

Comment:
avoid using index for keys where possible, if we don't have id, ignore

Reply:
In original json file there was not given ID, so I had two option add it by myself in json file or use index as a key.